pool/pool.go

```diff
@@ -895,12 +896,19 @@ type NodeParam struct {
 }
 
 // NewNodeParam creates NodeParam using parameters.
-func NewNodeParam(priority int, address string, weight float64) (prm NodeParam) {
+func NewNodeParam(priority int, address string, weight float64) (NodeParam, error) {
```

Comment:
how do u look to move this check to `pool.NewPool`?

Reply:
I don't mind, but how we should process the corner cases?
* One node and it is broken format
* Many nodes, and one/few in broken format. Some of nodes are ok

Reply:
see https://github.com/nspcc-dev/neofs-sdk-go/pull/471#discussion_r1267838896